use-cases/using-Intel-QAT-and-kata.md

```diff
@@ -0,0 +1,358 @@
+#Introduction
+
+Intel\* QuickAssist Technology (Intel\* QAT) provides hardware acceleration of security (cryptography) and compression. These instructions cover the steps for Clear Linux\* but can be adapted to any Linux\* distribution. Your distribution may already have the Intel QAT drivers, but it is likely they are out of date and do not contain the necessary user space components.  These instructions guide the user on how to download the kernel sources, compile kernel driver modules against those sources, and load them onto the host as well as preparing a specially built Kata Containers® kernel and custom Kata Containers rootfs.  
+
+##Helpful Links before starting
+[https://github.com/intel/QAT_Engine](https://github.com/intel/QAT_Engine)
+
+[https://01.org/intel-quickassist-technology](https://01.org/intel-quickassist-technology)
+
+[https://github.com/intel/intel-device-plugins-for-kubernetes/](https://github.com/intel/intel-device-plugins-for-kubernetes/)
+
+[https://dpdk-docs.readthedocs.io/en/latest/cryptodevs/qat.html](https://dpdk-docs.readthedocs.io/en/latest/cryptodevs/qat.html)
+
+
+##Steps to enable Intel QAT in Kata Containers
+Some of the steps only have to be done once. Some have to be done every reboot. Other steps are required only when the host kernel changes.
+
+
+1. [Script Variables](#script-variables)
```

Comment:
Please use dashes for table of contents entries. In fact I recommend you use the tool we have to generate the TOC automatically for you:

- https://github.com/kata-containers/tests/tree/master/cmd/check-markdown#generate-a-toc